src/components/Wizard/__stubs__/uischema.ts

```diff
@@ -0,0 +1,17 @@
+import { RowLayoutSchema } from '../../../types';
+
+export const uischema: RowLayoutSchema = {
+	type: 'RowLayout',
+	elements: [
+		{
+			type: 'Control',
+			label: 'Documentomschrijving',
+			scope: '#/properties/documentDescription',
+		},
+		{
+			type: 'Control',
+			label: 'Dummy date',
+			scope: '#/properties/dummyDate',
+		},
+	],
+};
```

Comment:
waarom constants wel named exporten, maar andere zaken, e.g. een FC niet. is niet consistent.
Ik zou dat rechttrekken met een linting rule. Ofwel via eer prefer default export of juist niet.